frontend_tests/casper_tests/14-drafts.js

```diff
@@ -175,6 +177,8 @@ casper.then(function () {
 
 casper.then(function () {
     casper.test.info('Finished reloading; now opening drafts again');
+    // Reloading into a narrow opens compose box automatically
+    casper.click("#compose_close");
```

Comment:
If reloading into a narrow opens the compose box automatically, shouldn't the `casper.click("#compose_close");` call happen before the `casper.reload()` in line 175?